lib/ProductOpener/Ingredients.pm

```diff
@@ -3325,7 +3326,7 @@ sub detect_allergens_from_text($) {
 
 			# allergens between underscores
 
-			# print STDERR "current text 1: $text\n";
+			print STDERR "current text 1: $text\n";
```

Comment:
We shouldn't use STDERR

Reply:
It's just for debugging/coding, that's why it's commented out. When debugging specific things in the code, I find it much easier than trying to get only the messages I want through log4perl.

Reply:
I don't think leaving it in the production code would be a good idea. It clutters the console

Reply:
Ah right I didn't intend to leave them, thanks for catching that, sorry I misunderstood the first time.